### PR TITLE
Tc5

### PR DESCRIPTION
**Pull request by vladstrazhnikov**

### CODE REVIEW

File: cypress/integration/TestCase5.spec.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        commonPage.clickButton(mainPage.menuButton, '/menu')
+
+        cy.log('Click on Iced teas link')
+        commonPage.clickButton(menuPage.icedTeasLink, '/menu/drinks/iced-teas')
+
+        cy.log('Click on Iced Black Tea link')
+        commonPage.clickButton(menuPage.icedBlackTeaLink, '/menu/product/457/iced')
+
+        cy.log('Click on Trenta size tea radiobutton')
+        cy.get(menuPage.trentaRadiobutton).check('Trenta', { force: true })
```
Comment (KirylParkhomenka) [inline cypress/integration/TestCase5.spec.js:24]: 1. What is the purpose of specifying 'Trenta' in `check()` method?
2. Remove `{ force: true }`
3. To specify element, which will not be covered, use xpath
4. Actually it's not a visible radiobutton for user. I'd better call it 'sizeOption'

File: cypress/pages/MenuPage.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export class MenuPage {
+    icedTeasLink = '[data-e2e="Iced Teas"]'
+    icedBlackTeaLink = '[data-e2e="Iced Black Tea"]'
+    trentaRadiobutton = '#Trenta'
+    flavorsButton = '[data-e2e="option-Flavors"]'
+    addBrownSugarSyropButton = '//div/div[1]/*[@class="stepper___1KL9Q"]//button[@class="quantityButton___7uYkS"]'
```
Comment (KirylParkhomenka) [inline cypress/pages/MenuPage.js:6]: Awful locator. Possible to make it shorter.

File: cypress/integration/TestCase5.spec.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        cy.get(menuPage.trentaRadiobutton).should('be.checked')
+
+        cy.log('Choose flavor')
+        menuPage.chooseFlavor(menuPage.addBrownSugarSyropButton)
+        cy.get(menuPage.flavorsButton).contains('1 pump(s) Brown Sugar Syrup')
+
+        cy.log('Choose juice')
+        cy.get(menuPage.juiceOptionsButton).click();
+        cy.get(menuPage.appleJuiceSelect).select('Light Apple Juice')
```
Comment (KirylParkhomenka) [inline cypress/integration/TestCase5.spec.js:33]: Please store these options in testData

File: cypress/integration/TestCase5.spec.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        cy.log('Click on Iced Black Tea link')
+        commonPage.clickButton(menuPage.icedBlackTeaLink, '/menu/product/457/iced')
+
+        cy.log('Click on Trenta size tea radiobutton')
+        cy.get(menuPage.trentaRadiobutton).check('Trenta', { force: true })
+        cy.get(menuPage.trentaRadiobutton).should('be.checked')
+
+        cy.log('Choose flavor')
+        menuPage.chooseFlavor(menuPage.addBrownSugarSyropButton)
+        cy.get(menuPage.flavorsButton).contains('1 pump(s) Brown Sugar Syrup')
```
Comment (KirylParkhomenka) [inline cypress/integration/TestCase5.spec.js:29]: Assertions in Cypress are performed using '[should](https://docs.cypress.io/api/commands/should#Syntax)'.
Need to replace `contains` with `should` everywhere.
Explanation why - you can find [here](https://stackoverflow.com/questions/50042670/cypress-test-is-contains-equivalent-to-shouldcontain).

**Review by KirylParkhomenka — CHANGES_REQUESTED** [review]

File: cypress/integration/TestCase5.spec.js (hunk 1 above)
Comment (vladstrazhnikov) [reply cypress/integration/TestCase5.spec.js:25]: As I understand I should replace trentaRadiobutton =  '#Trenta' with sizeOption =  '//*[@id="Trenta"]'?
And I tried different locator but without  { force: true } it does'n work
image

File: cypress/integration/TestCase5.spec.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+        cy.log('Click on Iced teas link')
+        commonPage.clickButton(menuPage.icedTeasLink, '/menu/drinks/iced-teas')
+
+        cy.log('Click on Iced Black Tea link')
+        commonPage.clickButton(menuPage.icedBlackTeaLink, '/menu/product/457/iced')
+
+        cy.log('Click on Trenta size tea radiobutton')
+        cy.xpath(menuPage.sizeOption).click()
+        cy.get(menuPage.trentaRadiobutton).should('be.checked')
```
Comment (KirylParkhomenka) [inline cypress/integration/TestCase5.spec.js:26]: Line 24 and 26 - radiobutton again?

File: cypress/pages/MenuPage.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+export class MenuPage {
+    icedTeasLink = '[data-e2e="Iced Teas"]'
+    icedBlackTeaLink = '[data-e2e="Iced Black Tea"]'
+    sizeOption =  '//*[@data-e2e="size-selector"]//fieldset/div[4]'
+    trentaRadiobutton = '#Trenta'
```
Comment (KirylParkhomenka) [inline cypress/pages/MenuPage.js:5]: Line 4 and 5 - selectors will be almost the same, but with small difference

**Review by KirylParkhomenka — CHANGES_REQUESTED** [review]

File: cypress/pages/MenuPage.js (hunk 6 above)
Comment (KirylParkhomenka) [reply cypress/pages/MenuPage.js:5]: Ok, use this one, but rename it

Comment (vladstrazhnikov) [reply cypress/pages/MenuPage.js:5]: or I can make something like '//*[@data-e2e="size-selector"]//*[@id="Trenta"]'
trentaItem is okay?

Comment (KirylParkhomenka) [reply cypress/pages/MenuPage.js:5]: no need, nothing will change

**Review by KirylParkhomenka — APPROVED** [review]

**Pull request merged**